operator/watchers/cilium_endpoint.go

```diff
@@ -163,3 +214,36 @@ func HasCE(ns, name string) (*cilium_api_v2.CiliumEndpoint, bool, error) {
 	cep := item.(*cilium_api_v2.CiliumEndpoint)
 	return cep, exists, nil
 }
+
+func endpointUpdated(cep *cilium_api_v2.CiliumEndpoint) {
+	if cep.Status.Networking == nil || cep.Status.Identity == nil || cep.GetName() == "" || cep.Namespace == "" {
+		return
```

Comment:
Would we ever hit this? I don't think a CEP will be propagated if its networking or identity or name is missing? @aanm please correct me if I'm wrong here.

Reply:
From CEB perspective, we are not taking in CEP.  If it doesn't have `networking or identity or name`.  if we pass this CEP to CEB, this will end up in expensive calls to ap-server.

Reply:
My point is that any CEP you got from watch would always have `networking` and `identity` set.

Reply:
If already set, that is good. But atleast one time i have noticed without values in `networking` field and eventually operator got crashed.

Reply:
@aanm Could we ever have a CEP without `networking` or `identity` field?

Reply:
Checked the code, I think `networking` might be empty, but none of `Name`, `Namespace` and `Identity` could be empty, can you updated the condition check?

Reply:
@Weil0ng let's not forget that older Cilium agents might be running and these fields will be empty so I think it's safe to check them as well.

Reply:
ah good point!